sktime/forecasting/base/_base.py

```diff
@@ -1545,9 +1553,14 @@ def _check_missing(metadata, obj_name):
         # end checking y
 
         # checking X
+
+        # check: if X is ignored by inner methods, pass None to them
+        if self.get_tag("ignores-exogeneous-X"):
```

Comment:
can you explain why this move is safe? We are not setting `X_scitype` anymore.

Reply:
This was already there before, I moved it to before the scitype checks which were unnecessary since `X` was being set to `None` anyway. 
`X_scitype` is being set to` None` in L-1610.